dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/visitor/JspVisitor.java

```diff
@@ -296,7 +296,7 @@ else if (jspElementType.equals(Constants.JSP_PARAMS_TYPE)) {
                         visitJspParamsStart(jspElement);
```

Comment:
We should update the copyright date range header to include 2023 and clean up the contributors.